extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -45,9 +45,9 @@ if (context.RunTest) {
 			let expectedActions;
 
 			if (process.platform === 'win32') {
-				expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Launch Profiler', 'Properties'];
```

Comment:
Was this intended to be in this review? Seems like it belongs to your other change to add the menu items

Reply:
oops yeah that should be in my other PR thanks